packages/tsconfig.json

```diff
@@ -245,6 +248,9 @@
     {
       "path": "./sources/kaiko"
     },
+    {
```

Comment:
Is this intentional?

Reply:
Yeah, it was accidentally not included in the original PR, so we should bring it in ASAP